_layouts/base.html

```diff
@@ -53,10 +53,10 @@
         {% include feed.html -%}
       {% endif -%}
     {% endcapture -%}
-    {{ plugin_output | replace: " />", ">" | replace: "/>", ">" -}}
+    {{ plugin_output | replace: " />", ">" | replace: "/>", ">" }}
```

Comment:
This change retains the line break preceding the next line in the generated markup.